gensrc/thrift/DataSinks.thrift

```diff
@@ -241,6 +241,7 @@ struct TIcebergTableSink {
     4: optional Types.TCompressionType compression_type
     5: optional bool is_static_partition_sink
     6: optional CloudConfiguration.TCloudConfiguration cloud_configuration
+    7: optional i64 target_max_file_size
```

Comment:
max_single_file_size ?

Reply:
this is a best-effort size. The file size could be larger than this value.

Reply:
got it
target_file_size is enough?